modules/order/src/Entity/OrderItem.php

```diff
@@ -261,6 +261,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDescription(t('The number of purchased units.'))
       ->setReadOnly(TRUE)
       ->setSetting('unsigned', TRUE)
+      ->setSetting('min', 0)
```

Comment:
Setting this prevents negative quantity. Kind of out of scope, but part of bigger problem.